### DISCUSSION TITLE
Keeping unit cell constant during gfnff geometry

### DISCUSSION THREAD

**Discussion by MasKlein-1** [grimme-lab/xtb#1042] [Q&A]
I was wondering if it is possible to do geometry optimizations under periodic boundary conditions using gfnff while not changing the unit cell parameters. From what I have gathered, gfnff is the only method that supports periodic boundary conditions natively but I have not seen anything that will allow me to fix the unit cell.

Thanks for the help!

**Comment by marcelmbn** [comment]
@Thomas3R might be able to help as he implemented GFN-FF with periodic boundary conditions.
